bootscore/bootscore
# Refactor header.php and enable Woo parts if WooCommerce is installed

In relation to https://github.com/bootscore/bootscore/issues/460

This PR uses `get_template_part` to get header buttons/widgets. WooCommerce related items load when WooCommerce is installed. No 2 different `header.php`'s needed anymore.

- No breaking changes to existing bootcommerce-child installations
- Allows us to delete `header.php` in bootcommerce-child


## How to test

Test in main-theme

- Remove `//` comments in `main.scss` line `4` to enable Woo styles
- Add following snippet to `functions.php` to enable Woo scripts if Woo is installed

```php
if ( class_exists( 'WooCommerce' ) ) {

  require get_template_directory() . '/woocommerce/woocommerce-functions.php';
  
  function my_function() {
    echo '<p class="alert alert-success">Woo installed</p>';
  }
  add_action('bs_after_primary', 'my_function', 5);

} else {

function my_function_2() {
    echo '<p class="alert alert-warning">Woo NOT installed</p>';
  }
  add_action('bs_after_primary', 'my_function_2', 5);

}
```

- Enable/disable WooCommerce plugin in backend and check the header in frontend
- Install a usual bootscore-child and override parts or entire header templates

@justinkruit 
- If you like it, merge it
- Is there a way to add `_bscore_woocommerce.scss` to the compiler and enable file only if WooCommerce is installed?
